data/yaml/venues.yaml

```diff
@@ -248,6 +248,9 @@ discomt:
   acronym: DiscoMT
   is_acl: true
   name: Workshop on Discourse in Machine Translation
+dmr:
```

Comment:
All the new venues lack an explicit `is_acl` property. I don't know whether the default is `true` or `false`, so stating it explicitly would be good imo.

Reply:
I removed all `false` instances.

Reply:
ah, okay. Missed that. In that case: LGTM, merge the way you like.